docs/communication/protocol/end-race.md

```diff
@@ -0,0 +1,82 @@
+[<- Back](../index.md)
+
+# Ending Race (By timeout or Force end)
```

Comment:
Please don't add dark colored boxes. Makes it difficult to read the text within
![image](https://user-images.githubusercontent.com/30839188/212523840-57f92d5b-2fbf-4b12-b793-4af9efb66541.png)

Reply:
Solved 041a145f872847b2d7afc7c52dbced6ad8ed4577